graphics/vtk/Portfile

```
@@ -9,13 +9,13 @@ PortGroup           compiler_blacklist_versions 1.0
 # Require C++11
 compiler.cxx_standard 2011
 
-# some older Clang say they support C++11 when they don't 
-# this is the same blacklisting as for jsoncpp, on which vtk depends. 
-compiler.blacklist-append {clang < 900} 
+# some older Clang say they support C++11 when they don't
+# this is the same blacklisting as for jsoncpp, on which vtk depends.
+compiler.blacklist-append {clang < 900}
 
 name                vtk
 version             8.2.0
-revision            2
+revision            3
```

This looks like a build fix only, in which case increasing the revision would not be warranted.
```suggestion
revision            2
```
Does this issue also affect the vtk5 port? If so maybe you would include the fix for that as well.